### ISSUE TITLE
Issue with Gradient-based Word Swap

### ISSUE THREAD

**Issue by jinyongyoo** [QData/TextAttack#209]
Adding 
```
if not self.model.emb_layer.embedding.weight.requires_grad:
            self.model.emb_layer.embedding.weight.requires_grad = True
```
to end of `__init__` of `WordSwapGradientBased` causes output for `hotflip` attacks to differ in test. 

Not sure why this happens.
